pkg/web/apps/apps_service_test.go

```diff
@@ -33,7 +35,7 @@ var (
 		CreateAppFunc: func(id string, appId string, name string) error {
 			return nil
 		},
-		GetAppByAppIDFunc: func(appID string) (*models.App, error) {
+		GetActiveAppByAppIDFunc: func(appID string) (*models.App, error) {
```

Comment:
Shows that you are removing the mocks as when you should just add the new ones. 

Followinng example.
```go
GetAppByAppIDFunc: func(appID string) (*models.App, error) {
			return nil, models.ErrNotFound
		},
```

Please, compare the mock tests with the master branch and ensure that you just added your new ones.

Reply:
Hi @camilamacedo86 good spot. This was done from the previous change when we thought we only needed one function. Changed now.